docs/ruleset_functions/get_HVAC_building_area_types_and_zones.md

```diff
@@ -0,0 +1,196 @@
+	- loop through each zone: `for zone in building_segment.zones:`
+		- add the zone to the building_area_types_with_total_area_and_zones_dict: `building_area_types_with_total_area_and_zones_dict[building_segment_HVAC_BAT]["ZONE_IDS"].append(zone.id)`
+			- loop through each space: `for space in zone.spaces:`
+				- add the space area to the building_area_types_with_total_area_and_zones_dict: `building_area_types_with_total_area_and_zones_dict[building_segment_HVAC_BAT]["AREA"] += space.floor_area`
+- we still need to deal with the possibility that there are spaces classified as OTHER_UNDETERMINED: `if OTHER_UNDETERMINED in building_area_types_with_total_area_and_zones_dict:`
+	- determine what the largest space type is: `predominate_HVAC_BAT = sorted(building_area_types_with_total_area_and_zones_dict.items(), key=lambda x: x[1]["AREA"], reverse = True)[0][0]`
+	- if RESIDENTIAL or OTHER_UNDETERMINED is the largest space area: `if predominate_HVAC_BAT == OTHER_UNDETERMINED or predominate_HVAC_BAT == RESIDENTIAL:`
+		- we re-classify all OTHER_UNDETERMINED spaces as OTHER_NON_RESIDENTIAL: `building_area_types_with_total_area_and_zones_dict[OTHER_NON_RESIDENTIAL] = building_area_types_with_total_area_and_zones_dict[OTHER_NON_RESIDENTIAL] or {"ZONE_IDS":[], "AREA":0, "CLASSIFICATION_SOURCE": building_area_types_with_total_area_and_zones_dict[OTHER_NON_RESIDENTIAL]["CLASSIFICATION_SOURCE"]}`
+		- add the zones from the OTHER_UNDETERMINED list to the OTHER_NON_RESIDENTIAL list: `building_area_types_with_total_area_and_zones_dict[OTHER_NON_RESIDENTIAL]["ZONE_IDS"] += building_area_types_with_total_area_and_zones_dict[OTHER_UNDETERMINED]["ZONE_IDS"]`
+		- add the area from the OTHER_UNDETERMIEND list to the OTHER_NON_RESIDENTIAL area: `building_area_types_with_total_area_and_zones_dict[OTHER_NON_RESIDENTIAL]["AREA"] += building_area_types_with_total_area_and_zones_dict[OTHER_UNDETERMINED]["AREA"]
+	- otherwise, we can add the are and zones from OTHER_UNDETERMINED to the predominate_HVAC_BAT: `else:`
+		- add the zones from the OTHER_UNDETERMINED list to the predominate_HVAC_BAT list: `building_area_types_with_total_area_and_zones_dict[predominate_HVAC_BAT]["ZONE_IDS"] += building_area_types_with_total_area_and_zones_dict[OTHER_UNDETERMINED]["ZONE_IDS"]`
+		- add the area from the OTHER_UNDETERMIEND list to the predominate_HVAC_BAT area: `building_area_types_with_total_area_and_zones_dict[predominate_HVAC_BAT]["AREA"] += building_area_types_with_total_area_and_zones_dict[OTHER_UNDETERMINED]["AREA"]
+	- remove the key OTHER_UNDETERMINED from building_area_types_with_total_area_and_zones_dict: `del building_area_types_with_total_area_and_zones_dict[OTHER_UNDETERMINED]`
+
+	 **Returns** `return building_area_types_with_total_area_and_zones_dict`  

```

Comment:
The dictionary that is returned by this function lists BAT, list of zones, and floor area for each segment. Shouldn't it then aggergate the matching values across the segments? For example, if building has two segments that both have the same BAT, should the returned dictionary include that BAT, list of all zones in the two segments, and the combined area of both segments?

Reply:
@mkarpman - the function follows your logic.  Lets' review on the call.

Reply:
discussed on RDS call and Karen clarified that the function follows the logic MK has laid out